packages/graphql-config-utilities/src/polyfills.ts

```diff
@@ -0,0 +1,16 @@
+// config.ts uses Array.flatmap
+import 'core-js/modules/es7.array.flat-map';
+
+declare global {
+  interface Array<T> {
+    flatMap<U, This = undefined>(
```

Comment:
rather than pull in typings or the `esnext.array` lib, i'm just defining `flatMap` inline (copy paste from `typescript/lib/lib.es2019.array.d.ts`)